src/components/socket.js

```diff
@@ -0,0 +1,14 @@
+import React, { createContext } from 'react';
+import { io, Socket } from 'socket.io-client';
+const url = ('http://localhost:8080');
+
+const socket = io(url), SocketContext = createContext(socket);
```

Comment:
when you open the page in the browser there is an error in console:

TypeError: (0 , c.io) is not a function
    at socket.js:5:16

see line 5 for that. you may want to try io.connect or something